### ISSUE TITLE
Multi-line comments & strings are incorrectly indented when Prettier is run.

### ISSUE THREAD

**Issue by christianmemije** [learningequality/kolibri#1800]
## Decription

* Multi-line comments are incorrectly indented when Prettier is run.
 
Example multi-line comment: 
```
   /**
   * A component for auto-generating a table of contents for a page. All
   * elements with an [id] attribute defined will be shown in the TOC.
   */
```

After multiple saves can end up like so:
```
  /**
           * A component for auto-generating a table of contents for a page. All
           * elements with an [id] attribute defined will be shown in the TOC.
            */
```

## Branch

* Develop

**Comment by rtibbles** [comment]
This is a known issue with prettier. Can you provide more context for the comment, is it inside an object declaration? This does not happen for all comment blocks.

**Comment by christianmemije** [comment]
Context:
https://github.com/learningequality/kolibri/blob/style_guide/kolibri/plugins/style_guide/assets/src/views/shell/table-of-contents.vue#L18

**Comment by jonboiser** [comment]
@christianmemije It might be because of the two-space offset we do inside of script tags. It does the same thing in the prettier play groung [link](https://prettier.io/playground/#%7B%22content%22%3A%22%20%2F**%5Cn%20%20*%20A%20component%20for%20auto-generating%20a%20table%20of%20contents%20for%20a%20page.%20All%5Cn%20%20*%20elements%20with%20an%20%5Bid%5D%20attribute%20defined%20will%20be%20shown%20in%20the%20TOC.%5Cn%20%20*%2F%5Cn%5Cn%2F%2F%20Two%20space%20offset.%20Notice%20first%20line%20starts%20in%20column%201%5Cn%20%20%2F**%5Cn%20%20%20*%20A%20component%20for%20auto-generating%20a%20table%20of%20contents%20for%20a%20page.%20All%5Cn%20%20%20*%20elements%20with%20an%20%5Bid%5D%20attribute%20defined%20will%20be%20shown%20in%20the%20TOC.%5Cn%20%20%20*%2F%5Cn%5Cn%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D)

**Comment by rtibbles** [comment]
As a workaround, I would recommend using // on multiple lines instead of the comment block for now.

**Comment by christianmemije** [comment]
A similar things happens here: 
https://github.com/learningequality/kolibri/blob/style_guide/kolibri/plugins/style_guide/assets/src/views/content/components/text-fields.vue#L149

**Comment by rtibbles** [comment]
Might be a weird interaction between the code I wrote to enforce our template linting and prettier.

**Comment by christianmemije** [comment]
Issue is within the indenting logic:
https://github.com/learningequality/kolibri/blob/develop/frontend_build/src/prettier-frontend.js#L38

**Comment by rtibbles** [comment]
Yeah, as @jonboiser pointed out, it is the interaction of prettier and the indentation - best solution is probably to dedent before prettifying!

**Comment by christianmemije** [comment]
Fixed in #1820
